frontend/src/css/app.sass

```diff
@@ -96,10 +96,66 @@ p, div
   @media (max-width: $breakpoint-sm-max)
     font-size: 3rem
```

Comment:
*arbitrary comment location*

Nit: seems we lost a space before the little icon:

Prod:

<img width="553" alt="Screen Shot 2023-05-16 at 11 35 23 AM" src="https://github.com/ScopeLift/umbra-protocol/assets/1479802/c2ad50ef-dc4d-4d18-9176-5542c22d7f6a">

Branch:

<img width="536" alt="Screen Shot 2023-05-16 at 11 35 48 AM" src="https://github.com/ScopeLift/umbra-protocol/assets/1479802/6cef2b42-533f-4845-9783-7c6eb9b1eb41">